CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+0.7.1 / 2021-09-16
+========================
+* added installation analytic event for nfs-provisioner ([#110](https://github.com/openebs/dynamic-nfs-provisioner/pull/110),[@mynktl](https://github.com/mynktl))
+
+
+0.7.0 / 2021-09-14
+========================
+* support to configure resource request and limit for NFS server resources ([#92](https://github.com/openebs/dynamic-nfs-provisioner/pull/92),[@mittachaitu](https://github.com/mittachaitu))
```

Comment:
IMO since we already specified in 0.6.1 we no need to repeat here

Reply:
we are comparing wrt to the old stable release, 0.6.0. 0.6.x is considered a patch release.